src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   extension) will now be set as the `Unset`, `Error`, or `Ok` string
   representation instead of the `0`, `1`, or `2` integer representation.
   ([#1579](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1579))
+* Metrics are obsolete and should't be used in production environment.
```

Comment:
I like the description you provide with the `Obsolete` attribute. I think providing the same info in the changelog could be useful. Something like:

> Metrics API/SDK support is in an experimental state and is not recommended for production use. All metric APIs have been marked with the `Obsolete` attribute. See https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501 for more information.

Reply:
thank you for reviewing. just updated!